src/model/migrations.ts

```diff
@@ -688,29 +685,22 @@ export default async function runMigrations() {
    Move favorites from local storage to react query persistent cache (AsyncStorage)
    */
   const v18 = async () => {
-    const favorites = await getGlobal('uniswapFavorites', undefined);
     const favoritesMetadata = await getGlobal(
       'uniswapjFavoritesMetadata',
```

Comment:
typo: extra j in `uniswapjFavoritesMetadata`

Reply:
ooo good one